dbms/src/Storages/DeltaMerge/SSTFilesToBlockInputStream.cpp

```diff
@@ -229,12 +237,15 @@ void SSTFilesToBlockInputStream::loadCFDataFromSST(
             region->insert(cf, TiKVKey(key.data, key.len), TiKVValue(value.data, value.len), DupCheck::AllowSame);
             reader->next();
             (*p_process_keys) += 1;
+            (*p_process_keys_bytes) += (key.len + value.len);
         }
         LOG_DEBUG(
             log,
-            "Done loading all kvpairs from [CF={}] [offset={}] [write_cf_offset={}] [region_id={}] [snapshot_index={}]",
+            "Done loading all kvpairs from [CF={}] [offset={}] [proc_size={}] [write_cf_offset={}] [region_id={}] "
+            "[snapshot_index={}]",
```

Comment:
```suggestion
            "Done loading all kvpairs, CF={} offset={} processed_size={} write_cf_offset={} region_id={} "
            "snapshot_index={}",
```
I think `processed_size` is better than `proc_size`